packages/web-components/src/toolbar/toolbar.spec.md

```diff
@@ -0,0 +1,88 @@
+# Toolbar
+
+## Component Description
+
+A toolbar is a container for grouping a set of controls, such as buttons, menubuttons, or checkboxes.
+
+When a set of controls is visually presented as a group, the toolbar role can be used to communicate the presence and purpose of the grouping to screen reader users. Grouping controls into toolbars can also be an effective way of reducing the number of tab stops in the keyboard interface.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/QfHyx6ApwFEcw5q4pM9CSZ/Toolbar?node-id=2351-6672&t=MuajdFxoCBjwlK4w-0)
+
+[Link to Fluent 2 Storybook](https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-toolbar--default)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr orientation: "horizontal" "vertical" | "horizontal"
+- @attr size: "small" "medium" "large" | "medium"
+
+### Outputs
+
+### Slots
+
+- label: Content which appears as the label for the toolbar.
+- start: Content which appears at the start of the toolbar, before the default slotted items.
+- default: Content which appears in the default location of the toolbar, left-aligned.
+- end: Content which appears at the end of the toolbar, right aligned.
+
+### CSS Variables
+
+None
+
+## Interactions
+
+Interactions are inherited from FAST Toolbar component.
+
+| Name             | Description                             |
+| ---------------- | --------------------------------------- |
+| `clickHandler`   | Sets the active index                   |
+| `focusinHandler` | Sets focus on element with active index |
+| `keydownHandler` | Handles keydown events on the toolbar   |
+
+## Accessibility
+
+- [x] Find the matching component through WCAG's patterns: https://www.w3.org/WAI/ARIA/apg/patterns/
+  - https://www.w3.org/WAI/ARIA/apg/patterns/toolbar.html
+- [x] Are there any accessibility elements unique to this component?
+- [x] List ARIA attributes
+  - [x] role="toolbar" in FAST
+  - [x] aria-orientation="vertical" in FAST
+- [ ] Does the component support 400% zoom?
+- [ ] What keyboard behaviors does the component support?
+  - [x] Tab : Moves focus to the next focusable element in the toolbar.
+  - [x] Shift + Tab : Moves focus to the previous focusable element in the toolbar.
+  - [x] Arrow keys : Moves focus to the next/previous focusable element in the toolbar.
```

Comment:
@chrisdholt Is this supposed to work in RTL?
The arrow navigation is not reversed in RTL (right arrows always navigates to the next item which is to the left in RTL)

Reply:
@chrisdholt -- Bug created and assigned: https://powerbi.visualstudio.com/Trident/_workitems/edit/1159913

Reply:
Noted elsewhere here but there's no bug - you need a story where the `direction` attribute is set to RTL. :)

Reply:
Added to new PR